internal/pkg/flink/internal/store/store.go

```diff
@@ -256,25 +256,23 @@ func NewStore(client ccloudv2.GatewayClientInterface, exitApplication func(), ap
 // Set properties default values if not set by the user
 // We probably want to refactor the keys names and where they are stored. Maybe also the default values.
 func (s *Store) propsDefault(propsWithoutDefault map[string]string) map[string]string {
-	properties := make(map[string]string)
+	properties := map[string]string{
+		config.ConfigKeyCatalog:          s.appOptions.GetEnvironmentId(),
+		config.ConfigKeyDatabase:         s.appOptions.GetKafkaClusterId(),
+		config.ConfigKeyOrgResourceId:    s.appOptions.GetOrgResourceId(),
+		config.ConfigKeyExecutionRuntime: "streaming",
```

Comment:
We needed this some time ago for the demo but I'm pretty sure we can delete this now.